src/si/ratio.rs

```diff
@@ -29,6 +32,51 @@ quantity! {
     }
 }
 
+/// Implementation of various stdlib inverse trigonometric functions
+#[cfg(feature = "std")]
+impl<U, V> Ratio<U, V>
+where
+    U: ::si::Units<V> + ?Sized,
+    V: ::num::Float + ::Conversion<V>,
+    radian: ::Conversion<V>,
```

Comment:
Adding `T = V::T` to the constraint should do the trick. See time.rs for an example.

https://github.com/iliekturtles/uom/blob/4fceafc98722464094aa7b658a8ff52d43944d00/src/si/time.rs#L89

Reply:
Thank you! Works like a charm.